### PR TITLE
Feat: Made Horizontal scrolling cards 

### PR DESCRIPTION
**Pull request by Meade-Reamy**

### CODE REVIEW

File: my-app/src/App.js
Comment (quasitonality) [inline my-app/src/App.js:1]: I would suggest putting all component-related elements within in the component itself (referring to the header element on line 22

**Review by quasitonality — CHANGES_REQUESTED** [review]
I think the header element on line 22 of App.js should be refactored into the RelatedCompare component, for better encapsulation

File: my-app/src/App.js
Comment (Meade-Reamy) [reply my-app/src/App.js:1]: Ok, moved it to inside the component

**Review by quasitonality — APPROVED** [review]
Looks good! 👍 I see there's a new react package, looking forward to hearing about it in stand-up

**Pull request merged**